test/srt/test_nightly_gsm8k_eval.py

```diff
@@ -26,19 +26,19 @@
     "deepseek-ai/DeepSeek-Coder-V2-Lite-Instruct": 0.85,
     "google/gemma-2-27b-it": 0.92,
     "meta-llama/Llama-3.1-70B-Instruct": 0.95,
-    "mistralai/Mixtral-8x7B-Instruct-v0.1": 0.63,
+    "mistralai/Mixtral-8x7B-Instruct-v0.1": 0.64,
     "Qwen/Qwen2-57B-A14B-Instruct": 0.86,
     "neuralmagic/Meta-Llama-3.1-8B-Instruct-FP8": 0.83,
     "neuralmagic/Mistral-7B-Instruct-v0.3-FP8": 0.54,
     "neuralmagic/DeepSeek-Coder-V2-Lite-Instruct-FP8": 0.84,
-    "neuralmagic/gemma-2-2b-it-FP8": 0.60,
+    "neuralmagic/gemma-2-2b-it-FP8": 0.50,
```

Comment:
don't change this

Reply:
@laixinn will help fix this

Reply:
ref https://github.com/sgl-project/sglang/issues/4324

Reply:
reverted